phoebe/backend/backends.py

```diff
@@ -14,7 +14,7 @@
 from phoebe.parameters import StringParameter, DictParameter, ArrayParameter, ParameterSet
 from phoebe.parameters.parameters import _extract_index_from_string
 from phoebe import dynamics
-from phoebe.backend import universe, horizon_analytic
+from phoebe.backend import universe, horizon_analytic, contacts_smoothing
```

Comment:
I'd probably call this something different than "contacts_smoothing." How about contact_envelopes or something similar?

Reply:
I'd be even more descriptive and make it `energy_transfer`. What that file does is describe ways to adjust the thermal properties of the meshes.